tidy3d/web/cli/app.py

```diff
@@ -0,0 +1,64 @@
+"""
+Commandline interface for tidy3d.
+"""
+import os.path
+from os.path import expanduser
+
+import click
+import requests
+import toml
+
+from tidy3d.web.config import DEFAULT_CONFIG
+
+TIDY3D_DIR = f"{expanduser('~')}/.tidy3d"
+CONFIG_FILE = TIDY3D_DIR + "/config"
+
+if os.path.exists(CONFIG_FILE):
+    with open(CONFIG_FILE, "r", encoding="utf-8") as f:
+        content = f.read()
+        config = toml.loads(content)
+        config_description = f"API Key[{config.get('apikey', '')}]"
+
+
+@click.group()
+def tidy3d_cli():
+    """
+    Tidy3d command line tool.
+    """
+
+
+@click.command()
+@click.option(
+    "--apikey", prompt=config_description if "config_description" in globals() else "API Key"
+)
+def configure(apikey):
+    """
```

Comment:
We use [numpy style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html) in Tidy3d. For consistency, can we change these and the other ones to numpy style? [Example](https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/web/webapi.py#L78-L100)